### PR TITLE
Workspace move cycle

### PR DESCRIPTION
**Pull request by apreiml**
On move workspace to direction: Try the farthest on the opposite direction if no workspace is found at given direction. 

This new behavior is closer to the i3 one now. I'd prefer it, because I'm using most of the time only one display connected to my notebook. With this addition I only need one shortcut to toggle a workspace between the two displays. I couldn't test this with more displays.

Depends on https://github.com/swaywm/wlroots/pull/1199

### CODE REVIEW

File: sway/commands/move.c [hunk 1]
```diff
@@ -37,19 +50,33 @@ static struct sway_container *output_in_direction(const char *direction,
 		{ "left", WLR_DIRECTION_LEFT },
 		{ "right", WLR_DIRECTION_RIGHT },
 	};
+
+	enum wlr_direction *direction = NULL;
```
Comment (RyanDwyer) [inline sway/commands/move.c]: Nitpick: This doesn't need to be a pointer.

File: sway/commands/move.c [hunk 2]
```diff
@@ -37,19 +50,33 @@ static struct sway_container *output_in_direction(const char *direction,
...
+
 	for (size_t i = 0; i < sizeof(names) / sizeof(names[0]); ++i) {
-		if (strcasecmp(names[i].name, direction) == 0) {
-			struct wlr_output *adjacent = wlr_output_layout_adjacent_output(
+		if (strcasecmp(names[i].name, direction_string) == 0) {
+			direction = &names[i].direction;
```
Comment (RyanDwyer) [inline sway/commands/move.c]: Another nitpick: You could add a `break` here to stop iterating the loop once you've found the direction.

**Comment by apreiml** [comment]
@RyanDwyer I changed the code according your suggestions.

**Comment by RedSoxFan** [comment]
Can you please rebase and force push? swaywm/wlroots#1199 has been merged so it should be good to merge after.

**Comment by apreiml** [comment]
@RedSoxFan done

**Pull request merged**